### PR TITLE
chore(build): add appveyor config

### PR DESCRIPTION
**Pull request by JaKXz**

### CODE REVIEW

**Comment by JaKXz** [comment]
@mattlewis92 FWIW I think I have a legit failure on Windows? It's in `istanbul-lib-sourcemaps/transformer`.

**Comment by mattlewis92** [comment]
@JaKXz I'm not actually a windows user so can't offer much help, I was just fixing a bug for a user of [one of my modules](https://github.com/mattlewis92/karma-coverage-istanbul-reporter)

**Comment by bcoe** [comment]
@JaKXz you are an admin now 👍 might be worth looking at how I've incrementally added windows tests here: https://github.com/conventional-changelog/conventional-changelog/blob/master/appveyor.yml

**Comment by bcoe** [comment]
@JaKXz friendly bump 👍 let me know anything you need help with to get this over the finish line.

**Comment by JaKXz** [comment]
@bcoe know of a way I can quickly debug this on my Mac? Unfortunately the cycles are really slow right now and I don't have access to a windows machine/image. I also don't know why that specific test is _actually_ failing as opposed to just timing out...

**Comment by bcoe** [comment]
@JaKXz https://github.com/bcoe/awesome-cross-platform-nodejs#virtualization

[ievms](https://github.com/xdissent/ievms) is a life saver.

File: lerna.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "packages": [
```
Comment (bcoe) [inline lerna.json]: I think you need to update the lerna version here too, if you're going to upgrade lerna.

Comment (JaKXz) [inline lerna.json]: Done!

**Comment by JaKXz** [comment]
Thanks for the review @bcoe but I'm really uncomfortable merging this while lerna is still so flaky on windows. Probably not lerna's fault but I'll cc @hzoo if he has any thoughts :)

**Comment by JaKXz** [comment]
@bcoe seems like v2.0.0-rc.3 is a lot more stable :) and I think the `--hoist` flag makes sense in general. What do you think?

**Comment by bcoe** [comment]
@JaKXz great work 👍

**Pull request merged**